LibVLCSharp.WPF/Themes/Generic.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:wf="clr-namespace:System.Windows.Forms;assembly=System.Windows.Forms"
+    xmlns:local="clr-namespace:LibVLCSharp.WPF">
+
+    <Style TargetType="{x:Type local:VideoView}">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type local:VideoView}">
+                    <Border Background="{TemplateBinding Background}"
```

Comment:
Any particular reason for why you added a `Border`?

Reply:
Yes, if we want that the BorderBrush, BorderThickness, ... properties of the Control works, we need to add a border in the ControlTemplate.

Reply:
Ok. 

Can we make it hidden or transparent though? My only concern with this PR is that it makes the view look different than the views on other platforms (adds a border). I think we should have the same looking result everywhere for now. But making it themeable/updateable is totally valid!

Reply:
Just talked with @mfkl, and I quite agree with him : All control should have the same behavior as much as possible (to be embedded in a X.F app for example).
Adding a border isn't problematic if its BorderThickness is 0 by default, but on other platforms, we don't have borders on the control.
Is that an issue to let users put a Border around the control?

Reply:
@jeremyVignelles For plattform consistency it's obvisously different. In most cases you have to create"border" or any other visual yourself, because they miss the functionality of XAML. It makes it possible, to create a different visual appearance for WPF only. On the other hand, as long as it is possible to deactivate it, it doesn't matter. Also i don't think most platform implementations support any theming after all, except WPF.

Reply:
>Adding a border isn't problematic if its BorderThickness is 0 by default
It's the case. By default, there is no border, no padding. I'm going to update the Example1 to remove the borders I added in order to test.

When you create a new control in WPF, there are these default properties (Padding, BorderBrush, ...) in the Control base class. In my opinion, when you create a custom control, these properties should work.

Reply:
@kakone : Fair enough, let's go for a border that is invisible by default.